### PR TITLE
enable master tests for python 3

### PR DESCRIPTION
**Pull request by tardyp**
but ignore the results..

so that we can have a status for master tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -62,11 +62,15 @@ matrix:
     # Specify SQLALCHEMY=latest to avoid errors installing.
     - python: "2.6"
       env: TWISTED=14.0.2 TESTS=trial_worker SQLALCHEMY=latest
-    - python: "2.6"
```
Comment (seankelly) [inline .travis.yml]: Why the deletion of this block?

Comment (tardyp) [inline .travis.yml]: its tested by bbtravis anyway. As travis is slower, I think we need to remove as much as possible stuff from travis

Comment (seankelly) [inline .travis.yml]: Makes sense. Any way to have bbtravis also test Python 3?

Comment (tardyp) [inline .travis.yml]: I will, but it takes more time, as I need to create a py3 capable docker image.

**Comment by rodrigc** [comment]
Apparently, [Mysql-python 1.2.5](https://pypi.python.org/pypi/MySQL-python/1.2.5) does not work on Python 3

According to the [Python 3 Wall of SuperPowers](https://python3wos.appspot.com/), if you look at the **py3k** link next to Mysql-python, it points to [mysqlclient](https://pypi.python.org/pypi/mysqlclient) which which is a PY3-compatible fork of Mysql-python

**Comment by tardyp** [comment]
@rodrigc I've got a bunch of Deferred object is unsubscriptable. Do I need to test against twisted trunk?

**Comment by rodrigc** [comment]
Twisted trunk has the same problem.  I posted this question: http://twistedmatrix.com/pipermail/twisted-python/2017-January/031091.html

**Comment by rodrigc** [comment]
@tardyp I have a fix for some of those Deferred object not subscriptable: https://github.com/buildbot/buildbot/pull/2661

**Pull request merged**